src/components/areaMenu/AposAreaMenuItem.vue

```diff
@@ -45,14 +47,14 @@ export default {
 
   methods: {
     click() {
-      this.$emit('click');
+      this.$emit("click");
     }
   }
-}
+};
 </script>
 
 <style lang="scss" scoped>
-@import '../../scss/_mixins';
+@import "../../scss/_mixins";
```

Comment:
Let's keep these single quotes.

Reply:
I'm seeing a ton of these. Is the linter complaining about the single quotes?

Reply:
Yes. Those were all picked up by eslint.

Reply:
From running `npm test` here? I'm unable to get the single quotes to throw an error. This is generally not our style.